timescaledb/tutorials/analyze-intraday-stocks/explore-stocks-data.md

```diff
@@ -290,13 +297,16 @@ Change `date` to see the candlesticks for another day.
 
 ## Resources
 
-* [Intraday stocks analysis on Github](https://github.com/timescale/examples/tree/master/analyze-intraday-stocks)
-* [Alpha Vantage API docs](https://www.alphavantage.co/documentation/)
+* [Intraday stocks analysis on Github][github-intraday-stocks]
+* [Alpha Vantage API docs][alpha-vantage-api]
 * [Pandas docs][pandas-docs]
 * [Plotly docs][plotly-docs]
-* [Timescale free trial](https://www.timescale.com/timescale-signup)
-* [Analyze cryptocurrency data](/tutorials/analyze-cryptocurrency-data/)
-
+* [Timescale free trial][timescale-signup]
+* [Analyze cryptocurrency data][analyze-cryptocurrency-data]
 
+[alpha-vantage-api]: https://www.alphavantage.co/documentation/
+[analyze-cryptocurrency-data]: /timescaledb/:currentVersion:/tutorials/analyze-cryptocurrency-data/
```

Comment:
not sure why Vale is complaining here, do we have a non-printable space or something? 🤔